### ISSUE TITLE
Replacing old Wiki markup in all files

### ISSUE THREAD

**Issue by the5thvolt** [aemodular/wiki#11]
Because the pages have been migrated from the old wiki which uses PmWiki markup there is now a confusing mix of PmWiki markup and new MarkDown markup.

It would be cool if someone could write a script which just goes over all files only once and changes all PmWiki markup to Markdown format.

EXCEPT: the Wikilink, image and embed ... that's still useful and mentioned in another ticket.

**Comment by TheGrandmother** [comment]
Fun fact related to the sizing of images!
https://docsify.js.org/#/helpers?id=resizing

**Comment by TheGrandmother** [comment]
This can be closed

**Comment by the5thvolt** [comment]
fixed by @TheGrandmother
